translations/messages.fr.xlf

```diff
@@ -0,0 +1,3063 @@
+        </target>
+      </trans-unit>
+      <trans-unit id="4418812710815829575" datatype="html">
+        <source>Failed, you have not captured the phalanx.</source>
+        <target>Raté, vous n&apos;avez pas capturé la phalange.</target>
+      </trans-unit>
+      <trans-unit id="7226802484619632640" datatype="html">
+        <source>You can only capture when 4 or more of your pieces are aligned, and it is not the case.</source>
+        <target>Une capture ne peut que se faire si 4 pièces de votre couleurs sont alignées, ce n&apos;est pas le cas.</target>
+      </trans-unit>
+      <trans-unit id="6918785733984182442" datatype="html">
+        <source>You must select a valid capture that contains 4 pieces or more.</source>
+        <target>Veuillez choisir une capture valide qui contient 4 pièces ou plus.</target>
+      </trans-unit>
+      <trans-unit id="6602326768713192004" datatype="html">
+        <source>There are still possible captures to be done.</source>
+        <target>Il vous reste des captures à effectuer.</target>
+      </trans-unit>
+      <trans-unit id="2434818181880718873" datatype="html">
+        <source>Pieces must be inserted in spaces on the edge of the board.</source>
+        <target>Les pièces doivent être placée sur une case du bord du plateau.</target>
+      </trans-unit>
+      <trans-unit id="7875793227562861246" datatype="html">
+        <source>You must select a valid placement direction.</source>
+        <target>Veuillez choisir une direction valide pour le déplacement.</target>
+      </trans-unit>
+      <trans-unit id="3111415789460197695" datatype="html">
+        <source>You must select a placement alongside the insertion space.</source>
+        <target>Veuillez choisir un placement avec une direction.</target>
+      </trans-unit>
+      <trans-unit id="1848361274892061756" datatype="html">
+        <source>You cannot place a piece on a complete line.</source>
+        <target>Veuillez effectuer un placement sur une ligne non complète.</target>
+      </trans-unit>
+      <trans-unit id="2854410959652074326" datatype="html">
+        <source>This piece belongs to two captures. Please select another piece in the capture you&apos;d like to take.</source>
+        <target>Veuillez sélectionner une autre case de la capture que vous souhaitez prendre, celle-ci appartient à deux captures.</target>
+      </trans-unit>
+      <trans-unit id="8407980394441393807" datatype="html">
+        <source>You must pick a capture.</source>
+        <target>Veuillez sélectionner une capture.</target>
+      </trans-unit>
+      <trans-unit id="3098052306255011996" datatype="html">
+        <source>You must select a destination space at a distance of 1 from the entrance.</source>
+        <target>Veuillez sélectionner une destination à une distance de 1 de l&apos;entrée.</target>
+      </trans-unit>
```

Comment:
Ce serait pas plus approprié et simple de dire "click on the arrow" ? 
là c'est du language de robot haha :D 
C'est ironique d'essayer de parler en langage plus humain et littéraire pour un site de jeu plutôt pour les matheux d'ailleurs :D

Reply:
vrai :D "You must click on an arrow to select your destination"

Reply:
Par contre c'est pour choisir la direction.
🖌️ 😩 😭 🍩